demo/common/assets.js

```diff
@@ -369,7 +369,9 @@ shakaAssets.testAssets = [
       .addFeature(shakaAssets.Feature.SUBTITLES)
       .addFeature(shakaAssets.Feature.WEBM)
       .addFeature(shakaAssets.Feature.OFFLINE)
-      .addLicenseServer('com.widevine.alpha', 'https://cwip-shaka-proxy.appspot.com/no_auth'),
+      .addLicenseServer('com.widevine.alpha', 'https://cwip-shaka-proxy.appspot.com/no_auth')
+      .setExtraConfig({drm: {advanced: {'com.widevine.alpha': {serverCertificateUri:
+        'https://storage.googleapis.com/wvmedia/cert/cert_license_widevine_com_uat.bin'}}}}),
```

Comment:
This isn't really needed as the CDM will request it for us anyway.  I'd suggest dropping the demo changes to make things simpler.

Reply:
Yep, it was just to have a working example with the serverCertificateUri. I could drop it or comment to explain this shouldn't be needed, but it is here to demonstrate the feature, WYT ?